src/error.rs

```diff
@@ -24,13 +24,13 @@ impl From<TonicError> for Error {
 
 impl From<TonicStatus> for Error {
     fn from(_error: TonicStatus) -> Self {
-        Error::GrpcError(GrpcError { status: _error })
+        Error::GrpcError(GrpcError { error: _error })
```

Comment:
do you have the reason to rename from status to error? `status` makes more sense to me if it includes gRPC status code and its details.

Reply:
You are right, `status` makes more sense. I changed to error because of the struct name `GrpcError`. I'll revert it to `status`. Thanks